src/linear_algebra/finite_dimensional.lean

```diff
@@ -111,6 +111,19 @@ begin
   cases exists_is_basis K V with s hs,
   exact ⟨s, hs, finite_of_linear_independent hs.1⟩
 end
+
+/-- In a finite dimensional space, there exists a finite basis. Provides the basis as a finset.
+This is in contrast to `exists_is_basis_finite`, which provides a set and a `set.finite`.
+-/
+lemma finite_dimensional.exists_is_basis_finset [finite_dimensional K V] :
```

Comment:
```suggestion
lemma exists_is_basis_finset [finite_dimensional K V] :
```

Reply:
Not sure if this is in a commit you forgot to push, but the linter is still complaining about this: https://github.com/leanprover-community/mathlib/runs/765849481#step:5:15